### DISCUSSION TITLE
Crashing Issue

### DISCUSSION THREAD

**Discussion by RileyWhite2** [Avnsx/fansly#72] [General]
Just started using the scrapper, works well enough, but it crashes/closes fairly consistently before it should be finished. Downloaded maybe 100 files each time I ran it, adding to itself each time, but it no longer seems to make any progress. Curious if it is a crashing issue, or a set up issue/hardware issue on my end

**Comment by Avnsx** [comment]
There's so much stuff to fix/improve, kinda didn't have the motivation though in the recent months.

I think this crashing issue is most likely, because of your RAM getting max'ed out. Someone pointed this out before already and it's already fixed on the python version here on github, but I didn't compile and release an update containing it yet.

**Reply by RileyWhite2** [reply]
Oh Alright, my bad, new to github and honestly am terrible at navigating it, I'll try and find the python version here on github

**Reply by Avnsx** [reply]
@RileyWhite2 You can download it here: https://github.com/Avnsx/fansly/archive/refs/heads/main.zip Then you need to install python from https://python.org/ and follow the set up guide for the raw python file: https://github.com/Avnsx/fansly#%EF%B8%8F-set-up
